src/main/resources/application.conf

```diff
@@ -3,27 +3,21 @@ database {
   config = hsqldb
 
   hsqldb {
-    url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
-    driver = "org.hsqldb.jdbcDriver"
-    slick.driver = "slick.driver.HsqldbDriver"
+    db.url = "jdbc:hsqldb:mem:memdb;shutdown=false;hsqldb.tx=mvcc"
+    db.driver = "org.hsqldb.jdbcDriver"
```

Comment:
must this prefix be `db`?  it's a bit confusing inside a vendor specific database block and feels specifically like JDBC info, which might be better within a block:

```
...
  jdbc {
    url = "..."
    driver = "..."
  }
...
```